### DISCUSSION TITLE
[back] docker desktop on mac 사용시 주의

### DISCUSSION THREAD

**Discussion by JuneParkCode** [3DPong/transcendence#3] [General]
컨테이너가 다 올라가고, 정상적으로 작동하고 있는데 커넥션이 성립을 안하는 문제가 있었습니다. 
``` text
ERROR [TypeOrmModule] Unable to connect to the database. Retrying (2)...
2023-03-12T14:30:30.685201043Z Error: connect ECONNREFUSED 127.0.0.1:5432
2023-03-12T14:30:30.685208293Z     at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1494:16)
```
이해가 안되서 뒤적거리니, 맥 도커 데스크탑에서 종종 있는 에러라네요.
typeorm 에서 db host 정할 때 `localhost` `127.0.0.1` 이 아닌 `host.docker.internal` 을 사용하면 해결할 수 있습니다.

https://stackoverflow.com/questions/33357567/econnrefused-for-postgres-on-nodejs-with-dockers
https://docs.docker.com/desktop/networking/#use-cases-and-workarounds

**Comment by kyu-baek** [comment]
저도 이러네요. 감사합니다.
